woocommerce/woocommerce-android
# Issue/2385 login tweaks

Fixes #2385 by doing the following: 
- Updating the theme-level `Woo.TextView` style to inherit from the material textView style so line breaking is handled properly.
- Overriding the `LoginTheme.Button.LoginOption` to apply the proper text button styles (Body1 + color_secondary). 

### Larger Text
The strange line-breaking issue is only noticeable if the **Display Size** on the test device is set to **Larger** like this:

<img src="https://user-images.githubusercontent.com/5810477/82109558-47dbdf80-96f4-11ea-9c2d-4503600ff74b.png" width="300"/>

Before | After
-- | --
![login-email-light-before](https://user-images.githubusercontent.com/5810477/82109590-75288d80-96f4-11ea-9cfe-4d123d057a21.png)|![login-email-light](https://user-images.githubusercontent.com/5810477/82109594-7954ab00-96f4-11ea-8a13-d7bfe6a3448e.png)
![login-email-dark-before](https://user-images.githubusercontent.com/5810477/82109596-7f4a8c00-96f4-11ea-949b-b21e2ab2be37.png)|![login-email-dark](https://user-images.githubusercontent.com/5810477/82109597-807bb900-96f4-11ea-975f-993e8416165e.png)

### Normal Text
Also notice the updated social login link colors and size:

Before | After
-- | --
![login-email-light-before-norm](https://user-images.githubusercontent.com/5810477/82109628-b02ac100-96f4-11ea-966d-86f720402139.png)|![login-email-light-norm](https://user-images.githubusercontent.com/5810477/82109630-b325b180-96f4-11ea-9689-12836b094ade.png)
![login-email-dark-before-norm](https://user-images.githubusercontent.com/5810477/82109631-b751cf00-96f4-11ea-8736-6c0116ef0f20.png)|![login-email-dark-norm](https://user-images.githubusercontent.com/5810477/82109632-b91b9280-96f4-11ea-9ede-1d08c2926150.png)

## To Test
Test the following on a device running less than API 28 and on a device running 28 or higher:
- Verify layout on "larger" display size setting
- Verify layout on normal display size setting


Update release notes:
